poprox-db/migrations/env.py

```diff
@@ -27,9 +27,9 @@
 # ... etc.
 
 db_user = os.environ.get("POPROX_DB_USER", "postgres")
-db_password = os.environ.get("POPROX_DB_PASSWORD", "")
+db_password = os.environ.get("POPROX_DB_PASSWORD", "thisisapoproxpassword")
```

Comment:
Not a big deal since these are just dev database credentials, but let's leave this out:
```suggestion
db_password = os.environ.get("POPROX_DB_PASSWORD", "")
```

Reply:
we should check that these changes don't break the tests. my _gut_ says that the tests don't currently change the terminal environment to plug-in the test-database (which I think is technically different from the dev-db, but is basically configured the same)